### PR TITLE
Add refresh path to proxy

### PR DESCRIPTION
**Pull request by soothing-rain**
/kind improvement

Signed-off-by: Yuchen Gao <yuchen.gao@zilliz.com>

### CODE REVIEW

File: internal/proxy/impl.go [hunk 1]
```diff
@@ -1407,7 +1408,8 @@ func (node *Proxy) LoadPartitions(ctx context.Context, request *milvuspb.LoadPar
 		zap.String("role", typeutil.ProxyRole),
 		zap.String("db", request.DbName),
 		zap.String("collection", request.CollectionName),
-		zap.Any("partitions", request.PartitionNames))
+		zap.Any("partitions", request.PartitionNames),
```
Comment (congqixia) [inline internal/proxy/impl.go:1411]: NIT: use `zap.Strings` instead of `zap.Any`

File: internal/proxy/impl.go [hunk 2]
```diff
@@ -453,7 +453,8 @@ func (node *Proxy) LoadCollection(ctx context.Context, request *milvuspb.LoadCol
 		zap.String("traceID", traceID),
 		zap.String("role", typeutil.ProxyRole),
 		zap.String("db", request.DbName),
-		zap.String("collection", request.CollectionName))
+		zap.String("collection", request.CollectionName),
+		zap.Bool("refresh mode", request.Refresh))
```
Comment (congqixia) [inline internal/proxy/impl.go:457]: NIT: camel style is recommended for log field name

**Review by congqixia — APPROVED** [review]
/lgtm

**Comment by soothing-rain** [comment]
rerun ut

**Pull request merged**